src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -42,34 +42,32 @@
      *
      * @var null|int[]
      */
-    private static $sortTypesOrder = null;
+    private $sortTypesOrder = null;
 
     /**
      * Array of supported sort types in configuration.
      *
      * @var string[]
      */
-    private static $supportedSortTypes = array(self::IMPORT_TYPE_CLASS, self::IMPORT_TYPE_CONST, self::IMPORT_TYPE_FUNCTION);
```

Comment:
But @keradus suggested to remove statics : https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2280#pullrequestreview-8316205

So what should I do? :)